src/pit.js

```diff
@@ -230,6 +234,8 @@ PIT.prototype.counter_write = function(i, value)
     {
         this.counter_next_low[i] ^= 1;
     }
+
+    this.bus.send("pcspeaker-update", this);
```

Comment:
Send only the values that are needed, not the whole object. The bus is designed for messages that can be send to a web worker, which works for arrays but won't for objects (at least not if the store functions).